mlir/test/Target/LLVMIR/vcix-rv64.mlir

```diff
@@ -0,0 +1,148 @@
+// RUN: mlir-translate --mlir-to-llvmir %s | FileCheck %s
```

Comment:
I'm not sure if `--split-input-file` is supported by `mlir-translate` but please try to add it as well.

Reply:
it does have this option. Will add it and delimiters to these tests